src/Tizen.Network.Stc/Tizen.Network.Stc/StcManager.cs

```diff
@@ -68,9 +73,11 @@ public static class StcManager
         /// <returns>The instance of the SafeStcHandle.</returns>
         /// <feature>http://tizen.org/feature/network.traffic_control</feature>
         /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="InvalidOperationException">Thrown when the method failed due to an invalid operation.</exception>
+        /// <exception cref="OutOfMemoryException">Thrown when there is not enough memory.</exception>
```

Comment:
X DO NOT explicitly throw OutOfMemoryException. This exception is to be thrown only by the CLR infrastructure.

https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/using-standard-exception-types#outofmemoryexception